csvapi/type_tester.py

```diff
@@ -36,6 +38,8 @@ def __init__(self):
         super(SirenSiret, self).__init__()
 
     def cast(self, d):
+        if d is None:
+            return d
         if is_valid_siret(d) or is_valid_siren(d):
```

Comment:
These functions don't crash for non-string arguments?

Reply:
I think the tests cover this. And I'd say every value come as a string since it's a CSV and we're trying to guess types.